php/containercontroller.class.inc

```diff
@@ -339,16 +337,18 @@ class ContainerController
     {
         if ($container->getId()) {
             $prevContainer = $this->dao->getInstanceFromId($container->getId());
-            $statusLabel = $this->dao->getStati()[$prevContainer->getStatusId()]['label'];
-            if ($statusLabel == 'Dispensed') {
-                // TODO: figure out what else should not be changed if the container
-                // is dispensed. This should eventually be blocked in the front-end
-                // as well. These validations can wait until more testing has 
-                // been done on the module.
-
-                if ($prevContainer->getTemperature() != $container->getTemperature()) {
-                    throw new \BadRequest("Temperature cannot be changed if Container
-                                           is Dispensed.");
+            if ($prevContainer) {
```

Comment:
@zaliqarosli were you running into an with the way things were here? Because if the `$container->getId()` exists, then the container instance derived from the ID should also exist.